### DISCUSSION TITLE
403 You don't have permission to access this resource.Server unable to read htaccess file, denying access to be safe

### DISCUSSION THREAD

**Discussion by Dany1410** [kimai/kimai#3350] [General]
Hello everybody,

as you can see in the title I have a problem accessing the .htaccess file.
I've been googleing the whole time but nothing really helps...
I assume (as the error tells) that this has to be a permission issue. I've been through the whole installation process on a Windows Machine and gave up in the end (the server wasn't able to access the index.php file).
Then I tried the installation procedure for Kimai on a new set up RHEL 8.4 Virtual Machine. This error response that the server is not able to access the .htaccess file seems to me to be similar to my last issue I had on the Windows Machine. 
I've set all the permissions according to the installation process. 
Something that looks strange to me is that I created a test.php file and placed it in /var/www/html to have a look, if somthing with php7.4 is wrong/missing. When I call http://localhost/test.php I get the exact same response as with calling my Kimai URL.
Restarting Apache didn't do it for me either...
I went through the whole installtion process except of the integration of the module libphp7.so in the httpd.conf file.

Can anybody relate or help me with my problem? 
Thank you in advance!

Best,
Daniel
![kimai_error log](https://user-images.githubusercontent.com/107126760/172642667-0b5febf2-6760-4d8f-8753-86354c9c13b3.PNG)

**Comment by kevinpapst** [comment]
Yeah, RHEL is  ... complicated. Read about SELinux. Easiest "solution" for the first step is to disable it and retry. 
This is all I will share, because it really has nothing to do with Kimai, but is a pure environment issue. 
Please ask in a Linux forum if SELinux is not the cause (what it is usually with RHEL).

**Reply by Dany1410** [reply]
Hey Kevin,

vielen Dank für die schnelle Antwort.

Alright, good to know. 
Thank you for your input. I didn't know about SELinux until now. I'll try to follow your suggestion and get back here to share the outcome.

**Comment by Dany1410** [comment]
Hey Kevin,

that was the right approach.
Kimai is working now.
Thank you very much!
![Kimai2](https://user-images.githubusercontent.com/107126760/173314383-0b4ac385-499a-4021-8f71-d2d6c45cf31b.PNG)
!
